thrust_forward_max: 1900
thrust_forward_min: 1600
thrust_turnr_max: 1900
thrust_turnr_min: 1600
thrust_turnl_max: 1400
thrust_turnl_min: 1100
pub_rate: 10
max_vel_x: 2
min_vel_x: 1
max_vel_theta: 2
min_in_place_vel_theta: 1

acc_lim_theta: 3.2
acc_lim_x: 2.5
acc_lim_y: 2.5
